src/main/java/com/odysseusinc/arachne/portal/config/WebSecurityConfig.java

```diff
@@ -255,10 +260,19 @@ protected void doFilterInternal(HttpServletRequest request, HttpServletResponse
         }
     }
 
+    @Autowired
+    AuthenticationSuccessHandler authenticationSuccessHandler;
+
     @Override
     protected void configure(HttpSecurity http) throws Exception {
-
         ExpressionUrlAuthorizationConfigurer<HttpSecurity>.ExpressionInterceptUrlRegistry reg = http
+                .requestCache().disable()  // TODO Remove before PR
```

Comment:
TODO?

Reply:
This was the plan. However, there is also a problem with the logout that has to be discussed first. As spring is caching requests based on their contents and is not aware about token invalidation and request caching filter is among the first ones in the chain.
Thus, if we load a page in one tab, then log out from different tab and reload the page in the original tab, it might will still load.